test/llvm-intrinsics/frexp.ll

```diff
@@ -0,0 +1,147 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis %t.rev.bc
+; RUN: FileCheck < %t.rev.ll %s --check-prefix=CHECK-LLVM
+
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir64-unknown-unknown"
+
+; CHECK-SPIRV: ExtInstImport [[#ExtInstSetId:]] "OpenCL.std"
+
+; CHECK-SPIRV: TypeInt [[#TypeInt:]] 32
```

Comment:
Do we want to make this CHECK-SPIRV-DAG in case the order changes in the future?

Reply:
Not sure about that - I don't think that types would be shuffled. 
I guess we should proceed as is and add DAG checks only if the ordering problem appears.